src/data/set/lattice.lean

```diff
@@ -279,7 +287,7 @@ theorem bInter_eq_Inter (s : set α) (t : Π x ∈ s, set β) :
   (⋂ x ∈ s, t x ‹_›) = (⋂ x : s, t x x.2) :=
 infi_subtype'
 
-theorem bInter_empty (u : α → set β) : (⋂ x ∈ (∅ : set α), u x) = univ :=
+@[simp] theorem bInter_empty (u : α → set β) : (⋂ x ∈ (∅ : set α), u x) = univ :=
```

Comment:
`lint` says that `simp` can prove this lemma.

Reply:
Oh, that explains why it was not marked as `simp`. I saw that lemma and thought it obviously deserved to be a `simp` lemma but I didn't think about this explanation.